### PR TITLE
#124 Add check name for lowercase checker

### PR DESCRIPTION
**Pull request by qelphybox**
Added check name for lowercase checker

#### ✔ Checklist:

- [x] This PR has been added to [CHANGELOG.md](https://github.com/mgrachev/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);
- [x] Tests for the changes have been added (for bug fixes / features);
- [x] Docs have been added / updated (for bug fixes / features).

Resolve: #124

### CODE REVIEW

File: src/checks/lowercase_key.rs [hunk 1]
```diff
@@ -8,7 +8,7 @@ pub(crate) struct LowercaseKeyChecker {
 impl Default for LowercaseKeyChecker {
     fn default() -> Self {
         Self {
-            template: String::from("The {} key should be in uppercase"),
+            template: String::from("LowercaseKey: The {} key should be in uppercase"),
```
Comment (mgrachev) [inline src/checks/lowercase_key.rs]: In the future I would like to have the ability to disable some checks by their names.
To do that, all checks should have the `name` field.
Please, move the check's name from the template to the special field.
Also, I would like to have a method to show the check's name in the message, for example:
```rust
impl LowercaseKeyChecker {
    fn message(&self, key: &str) -> String {
      // code...
    }
}
```

Comment (qelphybox) [reply src/checks/lowercase_key.rs]: Nice, i'll fix it

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### 🚀 Added
+- Added check name to lowercase check warning [#124](https://github.com/mgrachev/dotenv-linter/pull/131)
```
[the remark this answers is not in the document]
Comment (qelphybox) [reply CHANGELOG.md]: :+1:

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### 🚀 Added
 
 ### 🔧 Changed
+- Added check name to lowercase check warning [#124](https://github.com/mgrachev/dotenv-linter/pull/124) ([@qelphybox](https://github.com/qelphybox))
```
Comment (mgrachev) [inline CHANGELOG.md:11]:
```suggestion
- Added check name to lowercase check warning [#131](https://github.com/mgrachev/dotenv-linter/pull/131) ([@qelphybox](https://github.com/qelphybox))
```

File: src/checks/lowercase_key.rs [hunk 4]
```diff
@@ -19,14 +21,22 @@ impl Check for LowercaseKeyChecker {
         if key.to_uppercase() == key {
             None
         } else {
+            let warning_text = self.template.replace("{}", &key);
```
Comment (mgrachev) [inline src/checks/lowercase_key.rs:24]: You can move this line to the `message` function. For example:
https://github.com/mgrachev/dotenv-linter/blob/master/src/checks/unordered_keys.rs#L16

File: src/checks/lowercase_key.rs [hunk 5]
```diff
@@ -19,14 +21,22 @@ impl Check for LowercaseKeyChecker {
...
+
             Some(Warning::new(
                 line.clone(),
-                self.template.replace("{}", &key),
+                self.message(&warning_text),
```
Comment (mgrachev) [inline src/checks/lowercase_key.rs:28]: Please, run the `cargo fmt` command.
https://github.com/mgrachev/dotenv-linter/pull/131/checks?check_run_id=560584416

**Comment by mgrachev** [comment]
@qelphybox I have left some comments. Could you please fix them?

Also, I see that some tests are failed. Could you check that?
https://github.com/mgrachev/dotenv-linter/pull/131/checks?check_run_id=560584430

**Comment by mgrachev** [comment]
@qelphybox Thank a lot ❤️

**Comment by qelphybox** [comment]
Yeah, sorry I'm slow :(

**Pull request merged**